browser/src/control/Control.NotebookbarWriter.js

```diff
@@ -985,10 +989,11 @@ L.Control.NotebookbarWriter = L.Control.Notebookbar.extend({
 				'accessibility': { focusBack: false, combination: 'SD', de: null }
 			},
 			{
-				'id': 'format-FormatBulletsMenu',
```

Comment:
+1,
I hope we can drop at some point support for  '-' as a separator for menuId